packages/react-avatar/src/components/Avatar/Avatar.test.tsx

```diff
@@ -2,11 +2,42 @@ import * as React from 'react';
 import { isConformant } from '../../common/isConformant';
 import { Avatar } from './Avatar';
 import { render, screen } from '@testing-library/react';
+import { avatarClassNames } from './useAvatarStyles';
 
 describe('Avatar', () => {
   isConformant({
     Component: Avatar,
     displayName: 'Avatar',
+    testOptions: {
```

Comment:
this feels pretty verbose. is there a way to make this test more simple?

Reply:
I can think about it some more and I'm open to suggestions. The issue is that `Avatar` conditionally renders slots based on other slots -- `initials` is not rendered if there is an `icon`.

Reply:
Could we render all the avatars with these props? (pass as `requiredProps`)